server/routes/profile.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const router = express.Router();
+
+const auth = require("../middleware/auth");
+
+// Connect to profile controller
+const profileController = require("../controllers/profile");
+
+/* POST route to create profile within a user object - POST PROFILE operation. */

```

Comment:
The code is pretty clear what it does, these kind of comments are unnecessary.

Reply:
Good point. Thanks.